bloodview/src/graph.c

```diff
@@ -152,7 +155,7 @@ bool graph_create(unsigned idx, unsigned freq, uint8_t channel)
 
 		g->max = max;
 		g->x_step = freq / 500 + 1;
-		g->scale = Y_SCALE_DATUM / 8;
+		g->scale = Y_SCALE_DATUM >> 7;
```

Comment:
Any reason not to do "/ 128" here for readability? I'd only use shifts when thinking in bits.

Reply:
I've changed it to be less magical.